android/src/main/java/com/twiliorn/library/CustomTwilioVideoViewManager.java

```diff
@@ -181,6 +183,10 @@ public Map getExportedCustomDirectEventTypeConstants() {
                 ON_PARTICIPANT_REMOVED_AUDIO_TRACK, MapBuilder.of("registrationName", ON_PARTICIPANT_REMOVED_AUDIO_TRACK)
         ));
 
+        map.putAll(MapBuilder.of(
+                ON_DATATRACK_BINARY_MESSAGE_RECEIVED, MapBuilder.of("registrationName", ON_DATATRACK_BINARY_MESSAGE_RECEIVED)
```

Comment:
Maybe move this next to the `ON_DATATRACK_MESSAGE_RECEIVED` up on line 178, instead of having it in its own section.